server/schedule/operator/operator.go

```diff
@@ -48,16 +48,17 @@ type Cluster interface {
 
 // Operator contains execution steps generated by scheduler.
 type Operator struct {
-	desc        string
-	brief       string
-	regionID    uint64
-	regionEpoch *metapb.RegionEpoch
-	kind        OpKind
-	steps       []OpStep
-	currentStep int32
-	status      OpStatusTracker
-	stepTime    int64
-	level       core.PriorityLevel
+	desc         string
+	brief        string
+	regionID     uint64
+	regionEpoch  *metapb.RegionEpoch
+	kind         OpKind
+	steps        []OpStep
+	currentStep  int32
+	status       OpStatusTracker
+	stepTime     int64
+	level        core.PriorityLevel
+	UpdateMetric func()
```

Comment:
How about store `[]prometheus.Counter`? Later we can
```
for _, counter := range op.Counters {
  counter.Inc()
}
```

Reply:
Good idea

Reply:
PTAL